am-ui/webcomponents/search-webcomponent/src/app/search-form/form/form-field-control.service.ts

```diff
@@ -24,13 +24,11 @@ export class FormFieldControlService {
   /**
    * @description Funtion to transform array of fields into a FormGroup
    * @param {formFields} FormFieldBase[] - Array of category fields already transformed into FormFieldBase objects
+   * @param {offer} Boolean - Sets value of requestOrOffer for formcontrol
    * @return {FormGroup} form - The form with the data of the category fields
    */
-  toFormGroup(formFields: FormFieldBase<string>[]): FormGroup {
+  toFormGroup(formFields: FormFieldBase<string>[], offer: Boolean): FormGroup {
```

Comment:
for me it would be more natural to use isOffer for the boolean

Reply:
true, done